libflux/src/flux/parser/mod.rs

```diff
@@ -1111,10 +1111,18 @@ impl Parser {
     }
     fn parse_string_literal(&mut self) -> StringLit {
         let t = self.expect(TOK_STRING);
-        let value = strconv::parse_string(t.lit.as_str()).unwrap();
-        StringLit {
-            base: self.base_node_from_token(&t),
-            value,
+        match strconv::parse_string(t.lit.as_str()) {
+            Ok(value) => StringLit {
+                base: self.base_node_from_token(&t),
+                value,
+            },
+            Err(err) => {
+                self.errs.push(err);
+                StringLit {
+                    base: self.base_node_from_token(&t),
+                    value: "".to_string(),
+                }
```

Comment:
I can definitely see how this would've been easy to overlook. Glad the solution ended up being straightforward.